ui/src/app/exam/editor/sections/section-question.component.ts

```diff
@@ -119,7 +120,10 @@ export class SectionQuestionComponent {
                     })
                     .subscribe({
                         next: (resp) => {
-                            this.sectionQuestion = mergeDeepRight(this.sectionQuestion, resp) as ExamSectionQuestion;
+                            this.sectionQuestion = {
```

Comment:
Jostain syystä tuo merge ei riittänyt että UI päivittyy. En ihan ymmärrä miksi näin, mutta kopiointi sai näkymän päivittymään. Ehkä liittyy jotenkin tuon ramdan sisäiseen toimintaan jotenkin.

Lisäksi, jotta tuo päivitys näkyy parent-komponentille, piti vielä lähettää tollanen updated-eventti ja sille kuuntelija parent-komponentissa (ks. section.component.ts).

Reply:
Niin joo. Laitoin sen basequestioneditorin yhteyteen  myös ikäänkuin varmuudeksi, vaikka sanoit että siinä tuo toimisi. En testannut mutta otaksuin että siinä olisi sama ongelma.